website/docs/getting-started/setup/instance-configuration/README.mdx

```diff
@@ -0,0 +1,126 @@
+---
+description: >-
+  Appsmith ships with third-party services & configurations that improve the app
+  building experience. All services & configurations are entirely optional.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Instance Configuration
+
+Configuration of your Appsmith instance allows you to tailor it to your specific needs. This process includes setting up elements like environment variables, authentication, emails, SSL, and custom domain. Proper configuration ensures that your instance is running efficiently. 
+
+## Configure Docker installations
+
+To configure a docker installation, go to your installation folder and edit the `stacks/configuration/docker.env` file with the environment variables for the service.
+
+Remove `#` before the variables to ensure they're not commented.
```

Comment:
@sharat87 - Do we need this line? Can't we add parameters without `#`?

Reply:
Not strictly. Can be removed.